packages/cacti-plugin-copm-corda/Makefile

```diff
@@ -0,0 +1,39 @@
+WORKSPACE ?= $(shell realpath `pwd`/../../)
+
+build-local-deps:
+	cd $(WORKSPACE)/packages/cacti-copm-core/src/main/protos-kt && \
+	./gradlew build && \
+	cd $(WORKSPACE)/packages/cacti-copm-core && \
+	make corda-setup
+	mkdir -p localdeps && \
+	cp $(WORKSPACE)/packages/cacti-copm-core/src/main/protos-kt/build/libs/copm-corda-*jar localdeps 
+	cp $(WORKSPACE)/weaver/common/protos-java-kt/build/libs/protos-java-kt-*.jar localdeps
+	cp $(WORKSPACE)/weaver/sdks/corda/build/libs/weaver-sdk-corda-*.jar localdeps
+	cp $(WORKSPACE)/weaver/core/network/corda-interop-app/interop-workflows/build/libs/interop-workflows-*.jar localdeps
+	cp $(WORKSPACE)/weaver/core/network/corda-interop-app/interop-contracts/build/libs/interop-contracts-*.jar localdeps
+	cp $(WORKSPACE)/weaver/samples/corda/corda-simple-application/contracts-kotlin/build/libs/contracts-kotlin-*.jar localdeps
+	cp $(WORKSPACE)/weaver/samples/corda/corda-simple-application/workflows-kotlin/build/libs/workflows-kotlin-*.jar localdeps
```

Comment:
I can understand the rest, but why do you need `weaver/samples/...` as dependencies?

Reply:
The current corda interop chaincode require passing down the asset-specific CommandData instances for issue and burn, as well as getting the pledgeId.   This is something that could be avoided with asset type registration. 

Here are the asset-specific commands being passed to the chaincode in the corda CLI:  https://github.com/hyperledger-cacti/cacti/blob/27a24dd4579ef241df8d6bddf1cef30ee499b91d/weaver/samples/corda/corda-simple-application/clients/src/main/kotlin/com/cordaSimpleApplication/client/AssetTransferManager.kt#L116

Reply:
Corda interop app doesn't depend upon asset specific cordapp, it is designed to be general.
It just takes that command as a parameter, not needed to import or bundle the app cordapp with interop cordapp.

This makefile is moved to `src/test` folder, so I believe this is only for testing purpose and not a production based deployment? @jenniferlianne